src/main/java/com/dumbdogdiner/stickyapi/annotation/Untested.java

```diff
@@ -0,0 +1,8 @@
+/**
+ * Copyright (c) 2020 DumbDogDiner <a href="dumbdogdiner.com">&lt;dumbdogdiner.com&gt;</a>. All rights reserved.
```

Comment:
not a javadoc.

Reply:
It crashes the javadoc thing, if I remove those, then we need to change it so it inserts as a comment and not as javadoc.

Reply:
> It crashes the javadoc thing, if I remove those, then we need to change it so it inserts as a comment and not as javadoc.
cc: @Prouser123

Reply:
https://github.com/hierynomus/license-gradle-plugin reviewing from here

Reply:
there are a few other javadoc errors I'll fix in it too

Reply:
> It crashes the javadoc thing, if I remove those, then we need to change it so it inserts as a comment and not as javadoc. 


When you removed the link from the file, did you also remove it from the LICENSE_HEADER file?￼

Reply:
When it has the angle brackets (and not html escapes) javadoc gets mad. According to this and something it links, [https://github.com/hierynomus?tab=repositories], the fix should be:
```groovy
// License Plugin Options
license {
    header = project.file('LICENSE_HEADER')
    ext.year = Calendar.getInstance().get(Calendar.YEAR)
    mapping("java", "JAVAPKG_STYLE")
}
```
(I also made it put in the current year automatically)

The problem is that throws this error:
```
Execution failed for task ':licenseFormatMain'.
> The provided header type (javapkg_style) for java is invalid

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.

```

Reply:
fixed it wait for push